manifests/tekton/tasks/base/kustomization.yaml

```diff
@@ -2,5 +2,6 @@ bases:
   - cosa-init.yaml
   - cosa-build.yaml
   - cosa-buildextend.yaml
+  - cosa-upload-containers.yaml
```

Comment:
let's call this file `cosa-upload-images.yaml`

Reply:
Task renamed and moved.
Thanks @LorbusChris